service/service.go

```diff
@@ -60,6 +60,16 @@ type Service struct {
 	DeployedAt time.Time `hash:"-"`
 }
 
+// Instance represents a running instance of service.
+type Instance struct {
+	Sid         string
```

Comment:
What's the purpose of this sid? is it to be able to run an instance based on the sid? If this is the case it should be removed and the api/sdk should do the resolution and Instance only have a service hash

Reply:
It's for extra doc that we can send to cli on an instance create or delete action. I can move this and fetch it from service db. let's do it an another PR

Reply:
Yes please remove Sid.

Reply:
Also move the struct instance in its own package `instance`.